index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Birdwatching</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300%7CCinzel+Decorative:700"
+      rel="stylesheet"
+    />
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <header>
+      <h1>Birdwatching</h1>
+      <img src="dove.png" alt="a simple dove logo" />
+
+      <nav>
+        <ul>
+          <li><span>Home</span></li>
+          <li>
+            <a href="#">Get started</a>
+          </li>
+          <li><a href="#">Photos</a></li>
+          <li><a href="#">Gear</a></li>
+          <li><a href="#">Forum</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2>Welcome</h2>
+        <article>
+          <p>
+            Welcome to our fake birdwatching site. If this were a real site, it
+            would be the ideal place to come to learn more about birdwatching,
+            whether you are a beginner looking to learn how to get into birding,
+            or an expert wanting to share ideas, tips, and photos with other
+            like-minded people.
+          </p>
+        </article>
+
+        <article>
```

Comment:
This is still part of the previous article. You can just have it inside a separate `p` element